### ISSUE TITLE
The accuracy of KIE Model is 0 

### ISSUE THREAD

**Issue by TasneemVKhan** [PaddlePaddle/PaddleOCR#9395]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：OS Windows 11 
- 版本号/Version：Paddle：2.31.  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：python tools/train.py -c configs/kie/vi_layoutxlm/ser_vi_layoutxlm_xfund_zh.yml 
- 完整报错/Complete Error Message：

```
[2023/03/11 04:48:26] ppocr INFO: Architecture : 
[2023/03/11 04:48:27] ppocr INFO:     Backbone : 
[2023/03/11 04:48:27] ppocr INFO:         checkpoints : None
[2023/03/11 04:48:27] ppocr INFO:         mode : vi
[2023/03/11 04:48:27] ppocr INFO:         name : LayoutXLMForSer
[2023/03/11 04:48:27] ppocr INFO:         num_classes : 25
[2023/03/11 04:48:27] ppocr INFO:         pretrained : True
[2023/03/11 04:48:27] ppocr INFO:     Transform : None
[2023/03/11 04:48:27] ppocr INFO:     algorithm : LayoutXLM
[2023/03/11 04:48:27] ppocr INFO:     model_type : kie
[2023/03/11 04:48:27] ppocr INFO: Eval : 
[2023/03/11 04:48:27] ppocr INFO:     dataset : 
[2023/03/11 04:48:27] ppocr INFO:         data_dir : train_data/TC51/test/images
[2023/03/11 04:48:27] ppocr INFO:         label_file_list : ['train_data/TC51/test/test.json']
[2023/03/11 04:48:27] ppocr INFO:         name : SimpleDataSet
[2023/03/11 04:48:27] ppocr INFO:         transforms : 
[2023/03/11 04:48:27] ppocr INFO:             DecodeImage : 
[2023/03/11 04:48:27] ppocr INFO:                 channel_first : False
[2023/03/11 04:48:27] ppocr INFO:                 img_mode : RGB
[2023/03/11 04:48:27] ppocr INFO:             VQATokenLabelEncode : 
[2023/03/11 04:48:27] ppocr INFO:                 algorithm : LayoutXLM
[2023/03/11 04:48:27] ppocr INFO:                 class_path : train_data/XFUND/class_list_xfun_1.txt
[2023/03/11 04:48:27] ppocr INFO:                 contains_re : False
[2023/03/11 04:48:27] ppocr INFO:                 order_method : tb-yx
[2023/03/11 04:48:27] ppocr INFO:                 use_textline_bbox_info : True
[2023/03/11 04:48:27] ppocr INFO:             VQATokenPad : 
[2023/03/11 04:48:27] ppocr INFO:                 max_seq_len : 512
[2023/03/11 04:48:27] ppocr INFO:                 return_attention_mask : True
[2023/03/11 04:48:27] ppocr INFO:             VQASerTokenChunk : 
[2023/03/11 04:48:27] ppocr INFO:                 max_seq_len : 512
[2023/03/11 04:48:27] ppocr INFO:             Resize : 
[2023/03/11 04:48:27] ppocr INFO:                 size : [224, 224]
[2023/03/11 04:48:27] ppocr INFO:             NormalizeImage : 
[2023/03/11 04:48:27] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2023/03/11 04:48:27] ppocr INFO:                 order : hwc
[2023/03/11 04:48:27] ppocr INFO:                 scale : 1
[2023/03/11 04:48:27] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2023/03/11 04:48:27] ppocr INFO:             ToCHWImage : None
[2023/03/11 04:48:27] ppocr INFO:             KeepKeys : 
[2023/03/11 04:48:27] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'image', 'labels']
[2023/03/11 04:48:27] ppocr INFO:     loader : 
[2023/03/11 04:48:27] ppocr INFO:         batch_size_per_card : 8
[2023/03/11 04:48:27] ppocr INFO:         drop_last : False
[2023/03/11 04:48:27] ppocr INFO:         num_workers : 4
[2023/03/11 04:48:27] ppocr INFO:         shuffle : False
[2023/03/11 04:48:27] ppocr INFO: Global : 
[2023/03/11 04:48:27] ppocr INFO:     cal_metric_during_train : False
[2023/03/11 04:48:27] ppocr INFO:     distributed : False
[2023/03/11 04:48:27] ppocr INFO:     epoch_num : 200
[2023/03/11 04:48:27] ppocr INFO:     eval_batch_step : [0, 5]
[2023/03/11 04:48:27] ppocr INFO:     infer_img : ppstructure/docs/kie/input/zh_val_42.jpg
[2023/03/11 04:48:27] ppocr INFO:     kie_det_model_dir : None
[2023/03/11 04:48:27] ppocr INFO:     kie_rec_model_dir : None
[2023/03/11 04:48:27] ppocr INFO:     log_smooth_window : 10
[2023/03/11 04:48:27] ppocr INFO:     print_batch_step : 10
[2023/03/11 04:48:27] ppocr INFO:     save_epoch_step : 40
[2023/03/11 04:48:27] ppocr INFO:     save_inference_dir : None
[2023/03/11 04:48:27] ppocr INFO:     save_model_dir : ./output_3/ser_vi_layoutxlm_xfund_zh
[2023/03/11 04:48:27] ppocr INFO:     save_res_path : ./output_3/ser/xfund_zh/res
[2023/03/11 04:48:27] ppocr INFO:     seed : 42
[2023/03/11 04:48:27] ppocr INFO:     use_gpu : True
[2023/03/11 04:48:27] ppocr INFO:     use_visualdl : False
[2023/03/11 04:48:27] ppocr INFO: Loss : 
[2023/03/11 04:48:27] ppocr INFO:     key : backbone_out
[2023/03/11 04:48:27] ppocr INFO:     name : VQASerTokenLayoutLMLoss
[2023/03/11 04:48:27] ppocr INFO:     num_classes : 25
[2023/03/11 04:48:27] ppocr INFO: Metric : 
[2023/03/11 04:48:27] ppocr INFO:     main_indicator : hmean
[2023/03/11 04:48:27] ppocr INFO:     name : VQASerTokenMetric
[2023/03/11 04:48:27] ppocr INFO: Optimizer : 
[2023/03/11 04:48:27] ppocr INFO:     beta1 : 0.9
[2023/03/11 04:48:27] ppocr INFO:     beta2 : 0.999
[2023/03/11 04:48:27] ppocr INFO:     lr : 
[2023/03/11 04:48:27] ppocr INFO:         epochs : 200
[2023/03/11 04:48:27] ppocr INFO:         learning_rate : 5e-05
[2023/03/11 04:48:27] ppocr INFO:         name : Linear
[2023/03/11 04:48:27] ppocr INFO:         warmup_epoch : 2
[2023/03/11 04:48:27] ppocr INFO:     name : AdamW
[2023/03/11 04:48:27] ppocr INFO:     regularizer : 
[2023/03/11 04:48:27] ppocr INFO:         factor : 0.0
[2023/03/11 04:48:27] ppocr INFO:         name : L2
[2023/03/11 04:48:27] ppocr INFO: PostProcess : 
[2023/03/11 04:48:27] ppocr INFO:     class_path : train_data/XFUND/class_list_xfun_1.txt
[2023/03/11 04:48:27] ppocr INFO:     name : VQASerTokenLayoutLMPostProcess
[2023/03/11 04:48:27] ppocr INFO: Train : 
[2023/03/11 04:48:27] ppocr INFO:     dataset : 
[2023/03/11 04:48:27] ppocr INFO:         data_dir : train_data/TC51/train/images
[2023/03/11 04:48:27] ppocr INFO:         label_file_list : ['train_data/TC51/train/train.json']
[2023/03/11 04:48:27] ppocr INFO:         name : SimpleDataSet
[2023/03/11 04:48:27] ppocr INFO:         ratio_list : [1.0]
[2023/03/11 04:48:27] ppocr INFO:         transforms : 
[2023/03/11 04:48:27] ppocr INFO:             DecodeImage : 
[2023/03/11 04:48:27] ppocr INFO:                 channel_first : False
[2023/03/11 04:48:27] ppocr INFO:                 img_mode : RGB
[2023/03/11 04:48:27] ppocr INFO:             VQATokenLabelEncode : 
[2023/03/11 04:48:27] ppocr INFO:                 algorithm : LayoutXLM
[2023/03/11 04:48:27] ppocr INFO:                 class_path : train_data/XFUND/class_list_xfun_1.txt
[2023/03/11 04:48:27] ppocr INFO:                 contains_re : False
[2023/03/11 04:48:27] ppocr INFO:                 order_method : tb-yx
[2023/03/11 04:48:27] ppocr INFO:                 use_textline_bbox_info : True
[2023/03/11 04:48:27] ppocr INFO:             VQATokenPad : 
[2023/03/11 04:48:27] ppocr INFO:                 max_seq_len : 512
[2023/03/11 04:48:27] ppocr INFO:                 return_attention_mask : True
[2023/03/11 04:48:27] ppocr INFO:             VQASerTokenChunk : 
[2023/03/11 04:48:27] ppocr INFO:                 max_seq_len : 512
[2023/03/11 04:48:27] ppocr INFO:             Resize : 
[2023/03/11 04:48:27] ppocr INFO:                 size : [224, 224]
[2023/03/11 04:48:27] ppocr INFO:             NormalizeImage : 
[2023/03/11 04:48:27] ppocr INFO:                 mean : [123.675, 116.28, 103.53]
[2023/03/11 04:48:27] ppocr INFO:                 order : hwc
[2023/03/11 04:48:27] ppocr INFO:                 scale : 1
[2023/03/11 04:48:27] ppocr INFO:                 std : [58.395, 57.12, 57.375]
[2023/03/11 04:48:27] ppocr INFO:             ToCHWImage : None
[2023/03/11 04:48:27] ppocr INFO:             KeepKeys : 
[2023/03/11 04:48:27] ppocr INFO:                 keep_keys : ['input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'image', 'labels']
[2023/03/11 04:48:27] ppocr INFO:     loader : 
[2023/03/11 04:48:27] ppocr INFO:         batch_size_per_card : 8
[2023/03/11 04:48:27] ppocr INFO:         drop_last : False
[2023/03/11 04:48:27] ppocr INFO:         num_workers : 4
[2023/03/11 04:48:27] ppocr INFO:         shuffle : True
[2023/03/11 04:48:27] ppocr INFO: profiler_options : None
[2023/03/11 04:48:27] ppocr INFO: train with paddle 2.3.1 and device Place(gpu:0)
[2023/03/11 04:48:27] ppocr INFO: Initialize indexs of datasets:['train_data/TC51/train/train.json']
[2023-03-11 04:48:29,854] [    INFO] - Downloading https://bj.bcebos.com/paddlenlp/models/transformers/layoutxlm_base/sentencepiece.bpe.model and saved to /root/.paddlenlp/models/layoutxlm-base-uncased
[2023-03-11 04:48:29,855] [    INFO] - Downloading sentencepiece.bpe.model from https://bj.bcebos.com/paddlenlp/models/transformers/layoutxlm_base/sentencepiece.bpe.model
100% 4.83M/4.83M [00:17<00:00, 284kB/s]
[2023-03-11 04:48:49,773] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/tokenizer_config.json
[2023-03-11 04:48:49,774] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/special_tokens_map.json
[2023/03/11 04:48:50] ppocr INFO: Initialize indexs of datasets:['train_data/TC51/test/test.json']
[2023-03-11 04:48:50,416] [    INFO] - Already cached /root/.paddlenlp/models/layoutxlm-base-uncased/sentencepiece.bpe.model
[2023-03-11 04:48:50,972] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/tokenizer_config.json
[2023-03-11 04:48:50,972] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/layoutxlm-base-uncased/special_tokens_map.json
[2023-03-11 04:48:51,390] [    INFO] - Downloading https://bj.bcebos.com/paddlenlp/models/transformers/vi-layoutxlm-base-uncased/model_state.pdparams and saved to /root/.paddlenlp/models/vi-layoutxlm-base-uncased
[2023-03-11 04:48:51,390] [    INFO] - Downloading model_state.pdparams from https://bj.bcebos.com/paddlenlp/models/transformers/vi-layoutxlm-base-uncased/model_state.pdparams
100% 1.04G/1.04G [01:55<00:00, 9.62MB/s]
W0311 04:50:48.624604  2984 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 12.0, Runtime API Version: 10.2
W0311 04:50:49.076179  2984 gpu_resources.cc:91] device: 0, cuDNN Version: 8.7.
[2023/03/11 04:50:56] ppocr INFO: train dataloader has 2 iters
[2023/03/11 04:50:56] ppocr INFO: valid dataloader has 2 iters
[2023/03/11 04:50:56] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 5 iterations
[2023/03/11 04:51:07] ppocr INFO: epoch: [1/200], global_step: 2, lr: 0.000006, loss: 3.339764, avg_reader_cost: 0.55090 s, avg_batch_cost: 1.09762 s, avg_samples: 1.5, ips: 1.36659 samples/s, eta: 0:36:24
[2023/03/11 04:51:55] ppocr INFO: save model in ./output_3/ser_vi_layoutxlm_xfund_zh/latest
[2023/03/11 04:52:03] ppocr INFO: epoch: [2/200], global_step: 4, lr: 0.000019, loss: nanxxx, avg_reader_cost: 5.37815 s, avg_batch_cost: 5.49248 s, avg_samples: 1.5, ips: 0.27310 samples/s, eta: 1:48:44
[2023/03/11 04:52:25] ppocr INFO: save model in ./output_3/ser_vi_layoutxlm_xfund_zh/latest
eval model:: 100% 2/2 [00:12<00:00,  6.16s/it]
[2023/03/11 04:52:44] ppocr INFO: cur metric, precision: 0.0, recall: 0.0, hmean: 0.0, fps: 21.776766763537584
[2023/03/11 04:53:29] ppocr INFO: save best model is to ./output_3/ser_vi_layoutxlm_xfund_zh/best_accuracy
[2023/03/11 04:53:29] ppocr INFO: best metric, hmean: 0.0, precision: 0.0, recall: 0.0, fps: 21.776766763537584, best_epoch: 3
[2023/03/11 04:53:30] ppocr INFO: epoch: [3/200], global_step: 6, lr: 0.000031, loss: nanxxx, avg_reader_cost: 2.75983 s, avg_batch_cost: 2.88289 s, avg_samples: 1.5, ips: 0.52031 samples/s, eta: 1:43:40
[2023/03/11 04:53:52] ppocr INFO: save model in ./output_3/ser_vi_layoutxlm_xfund_zh/latest
[2023/03/11 04:54:00] ppocr INFO: epoch: [4/200], global_step: 8, lr: 0.000044, loss: nanxxx, avg_reader_cost: 2.84651 s, avg_batch_cost: 2.95761 s, avg_samples: 1.5, ips: 0.50717 samples/s, eta: 1:41:30
[2023/03/11 04:54:21] ppocr INFO: save model in ./output_3/ser_vi_layoutxlm_xfund_zh/latest
[2023/03/11 04:54:28] ppocr INFO: epoch: [5/200], global_step: 10, lr: 0.000049, loss: nanxxx, avg_reader_cost: 2.71508 s, avg_batch_cost: 2.82572 s, avg_samples: 1.5, ips: 0.53084 samples/s, eta: 1:39:09
eval model:: 100% 2/2 [00:05<00:00,  2.61s/it]
[2023/03/11 04:54:34] ppocr INFO: cur metric, precision: 0.0, recall: 0.0, hmean: 0.0, fps: 23.101823887339126
[2023/03/11 04:54:59] ppocr INFO: save best model is to ./output_3/ser_vi_layoutxlm_xfund_zh/best_accuracy
[2023/03/11 04:54:59] ppocr INFO: best metric, hmean: 0.0, precision: 0.0, recall: 0.0, fps: 23.101823887339126, best_epoch: 5
[2023/03/11 04:55:23] ppocr INFO: save model in ./output_3/ser_vi_layoutxlm_xfund_zh/latest
[2023/03/11 04:55:29] ppocr INFO: epoch: [6/200], global_step: 12, lr: 0.000049, loss: nanxxx, avg_reader_cost: 2.82946 s, avg_batch_cost: 2.93744 s, avg_samples: 1.5, ips: 0.51065 samples/s, eta: 1:38:02

```

**Comment by huangjun12** [comment]
loss出nan了，调低一下学习率试试

**Comment by TasneemVKhan** [comment]
My learning rate is 0.00005. It's already very low.  I changed it to 0.000005. I'm still getting the same result.

**Comment by NaumanHSA** [comment]
@TasneemVKhan I'm facing the same issue. Have you done any research on this, anything useful?

**Comment by TasneemVKhan** [comment]
> 

I haven't been able to find anything regarding this issue. I have tried various methods but hasn't been successful.

**Comment by NaumanHSA** [comment]
@TasneemVKhan I tried training on Ubuntu 20 with the same dataset, and same environment with RTX 3080ti. It worked for me, I'm getting some accuracy now.

**Comment by TasneemVKhan** [comment]
I'm using my own dataset using the FUNSD format.

**Comment by TasneemVKhan** [comment]
I'm using UBIAI for annotation. I'll check out whatever you have said and let you know.

**Comment by munish0838** [comment]
Hi @TasneemVKhan, could you share your FUNSD dataset or training process if you were able to train model for RE task

**Comment by Rishi-NF** [comment]
@munish0838 Have a found a way to train a kie model? any information will be greatly appreciated
